utils/build-script

```diff
@@ -74,47 +74,57 @@ class HostSpecificConfiguration(object):
     def __init__(self, host_target, invocation):
         """Initialize for the given `host_target`."""
 
-        # Compute the set of deployment targets to configure/build.
         args = invocation.args
+
+        # [*] Compute the list of deployment targets to configure.
```

Comment:
None. It's a major step in the workflow (as opposed to all of the minor ones) so it's just for emphasis.